pfsspy/__init__.py

```diff
@@ -512,6 +513,74 @@ def _common_b(self):
         self._common_b_cache = br, bs, bp, Sbr, Sbs, Sbp
         return self._common_b_cache
 
+    def get_Bvec(self,skycoord,out_type="cartesian") :
+        """
+        Input
+        =====
```

Comment:
These should be dashes `-` (and similarly below)

Reply:
The docstrings all need formatting to match the other docstrings, which are in the numpydoc format: https://numpydoc.readthedocs.io/en/latest/format.html - I'm happy to do this if it's easier?

Reply:
Thanks David, please go ahead and edit the docstrings.

Reply:
See latest commits - I think I have conformed to the numpy standards and the docs build looks good